### PR TITLE
Folasade/imp 27 profile frontend

### PR DESCRIPTION
**Pull request by foloye**
### Description

[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

Frontend portion of the portion tab is completed.  There is currently no frontend for the reset password button.

- Styling for main settings tab is done
- Styling for edit name screen is done
- Styling for edit address screen is done
- Styling for delete account is to be designed and developed

### Screenshots

[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy!"
Screenshot 2023-11-27 at 7 58 29 PM
Screenshot 2023-11-27 at 7 58 55 PM
Screenshot 2023-11-27 at 7 58 41 PM

## How to review

[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

It can reviewed all at once. I would like some feedback on how the input fields work in certain edge cases especially for the edit address fields. There's no error handling for the address, but this screen gave me the most bugs while implementing

## Next steps
Next step is to build out the frontend for the delete account screen.

[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### Online sources

[//]: # 'Optional - copy links to any tutorials or documentation that was useful to you when working on this PR'

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @stephaniewong2

### CODE REVIEW

File: src/app/(BottomTabNavigation)/Profile/EditAddress/index.tsx [hunk 1]
```diff
@@ -1,18 +1,28 @@
...
 function EditNameScreen() {
-  const [streetName, setStreetName] = useState<string>();
-  const [usState, setUsState] = useState<string>();
-  const [city, setCity] = useState<string>();
-  const [zip, setZip] = useState<string>();
+  const [streetName, setStreetName] = useState<string>('');
+  const [usState, setUsState] = useState<string>('');
+  const [city, setCity] = useState<string>('');
+  const [zip, setZip] = useState<string>('');
+  const [displayName, setName] = useState<boolean>(true);
```
Comment (stephaniewong2) [inline src/app/(BottomTabNavigation)/Profile/EditAddress/index.tsx]: Let's define separate display names for each input field

File: src/app/(BottomTabNavigation)/Profile/EditAddress/index.tsx [hunk 2]
```diff
@@ -22,44 +32,93 @@ function EditNameScreen() {
...
+      </TouchableOpacity>
+      <Text style={styles.instructionText}>Edit account details</Text>
+
+      <View style={styles.inputBox}>
+        <AuthInput
+          input={streetName}
+          setInput={setStreetName}
+          defaultValue="Street Name"
+          isPassword={false}
+          displayInput={displayName}
```
Comment (stephaniewong2) [inline src/app/(BottomTabNavigation)/Profile/EditAddress/index.tsx]: display name should be unique for each `AuthInput`!

File: src/app/(BottomTabNavigation)/Profile/EditName/index.tsx [hunk 3]
```diff
@@ -1,44 +1,66 @@
...
   getCurrentUserInfo,
   updateCurrUserName,
 } from '../../../../supabase/queries/auth';
-import { User, userInstance } from '../../../../types/types';
 
 function EditNameScreen() {
-  const [currSession, setCurrSession] = useState<User>(userInstance);
-  const [fullName, setFullName] = useState<string>();
+  const [fullName, setFullName] = useState<string>('');
+  const [displayFullName, setDisplayFullName] = useState<boolean>(true);
```
Comment (stephaniewong2) [inline src/app/(BottomTabNavigation)/Profile/EditName/index.tsx:15]: Let's define a `placeholder` state variable and use it for `placeholder` and `setPlaceholder` in the `AuthInput`!

File: src/app/(BottomTabNavigation)/Profile/index.tsx [hunk 4]
```diff
@@ -1,36 +1,127 @@
...
+import SignOut from '../../../../assets/sign-out.svg';
+import CasesHeader from '../../../Components/CasesHeader/CasesHeader';
+import {
+  getCurrentUserInfo,
+  signOutUser,
+} from '../../../supabase/queries/auth';
+import { User, userInstance } from '../../../types/types';
 
 function ProfileScreen() {
+  const [currSession, setCurrSession] = useState<User>(userInstance);
```
Comment (stephaniewong2) [inline src/app/(BottomTabNavigation)/Profile/index.tsx:22]: When we navigate back to this screen after updating a user's information, the fields are not updated. I think we can address this when we implement Auth Context though.

**Review by stephaniewong2 — CHANGES_REQUESTED** [review]
SO GOOD!! Just few minor changes!

File: src/app/(BottomTabNavigation)/Profile/index.tsx (hunk 4 above)
Comment (foloye) [reply src/app/(BottomTabNavigation)/Profile/index.tsx:22]: yes I think currSession stuff will be addressed when I do auth context 🏄🏾‍♀️

File: src/app/(BottomTabNavigation)/Profile/index.tsx [hunk 5]
```diff
@@ -1,36 +1,127 @@
...
+              <Text style={styles.textElements}>Street address</Text>
+            </View>
+            <TouchableOpacity
+              onPress={() => router.push('/Profile/EditAddress')}
+            >
+              <Pencil style={styles.edit} />
+            </TouchableOpacity>
+          </View>
+          <Text style={styles.userText}>
+            {currSession.streetName +
```
Comment (stephaniewong2) [inline src/app/(BottomTabNavigation)/Profile/index.tsx:75]: Let's display the address as 
Street Name 
City, State Zip
